Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1858,6 +1864,19 @@ + (void)maybePerformActions:(NSArray *)whenConditions
     }
 }
 
++ (NSNumber *)fetchCountDownForContext:(LPActionContext *)actionContext
+{
+    NSDictionary *messageConfig = [LPVarCache sharedCache].messageDiffs[actionContext.messageId];
+    NSNumber *countdown = messageConfig[@"countdown"];
+    if (actionContext.isPreview) {
+        countdown = @(5.0);
+    }
+    if (![countdown.class isSubclassOfClass:NSNumber.class]) {
+        countdown = @(1.0);
```

Comment:
how are we assigning these numbers?

Reply:
Thanks for noting this one, i forgot to remove this, we dont need this here.
Here is the reason for it though.
Thats a great question. I had a discussion with Arash on this one. What if there is no value for "countdown" , what does the web set as default. And looks like Web defaults it at 1.

I had later tested with Yilong on this one. And that we are setting defaults on server side to either 1 or 24 hours. So, i dont think , i need this extra check here.